components/database/berkeleydb/berkeleydb.p5m

```diff
@@ -21,22 +21,49 @@
 
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2018, Michal Nowak
 #
 
-<transform file path=usr.*/man/.+ -> default mangler.man.stability committed>
 <transform file path=usr/share/doc/bdb/(.*) -> set action.hash usr/docs/%<\1> >
-set name=pkg.fmri \
-    value=pkg:/database/berkeleydb-5@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
 set name=pkg.summary value="Oracle Berkeley DB"
 set name=pkg.description \
     value="Berkeley DB is a computer software library that provides a high-performance embedded database for key/value data."
-set name=info.classification \
-    value=org.opensolaris.category.2008:System/Databases
+set name=info.classification value=org.opensolaris.category.2008:System/Databases
 set name=info.keyword value=bdb
-set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
-set name=org.opensolaris.arc-caseid value=PSARC/2010/300
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license berkeleydb.license license="berkeleydb license"
+file path=usr/docs/LICENSE
+
+# Keep v5.1 compat links
+link path=usr/lib/libdb-5.1.so target=libdb-5.3.so
```

Comment:
Sorry, but why do we need them?

Reply:
I thought this is what you asked me to preserve when you said "preserve old -5.1 compatibility symlinks".

Reply:
Yes, sure. I'm just not sure, if we need them or not.

Reply:
But I miss something, you deliver not only 5.1-compat symlinks in this section, and some actions are included twice, like 'link path=usr/lib/libdb.so target=libdb-5.3.so'